tools/points-service/main.go

```diff
@@ -0,0 +1,758 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"log/slog"
+	"math/big"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	avs "github.com/primev/mev-commit/contracts-abi/clients/MevCommitAVS"
+	middleware "github.com/primev/mev-commit/contracts-abi/clients/MevCommitMiddleware"
+	validatoroptinrouter "github.com/primev/mev-commit/contracts-abi/clients/ValidatorOptInRouter"
+	vanillaregistry "github.com/primev/mev-commit/contracts-abi/clients/VanillaRegistry"
+	vault "github.com/primev/mev-commit/contracts-abi/clients/Vault"
+	config "github.com/primev/mev-commit/contracts-abi/config"
+	"github.com/primev/mev-commit/x/contracts/events"
+	"github.com/primev/mev-commit/x/contracts/events/publisher"
+	"github.com/urfave/cli/v2"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// ~216000 blocks is roughly one month on Ethereum (~2s block time)
+var blocksInOneMonth = int64(216000)
+
+var (
+	rwLock sync.RWMutex
+
+	createTableValidatorRecordsQuery = `
+    CREATE TABLE IF NOT EXISTS validator_records (
```

Comment:
Use only one indentation for all. The create table queries use different indentation than the other queries.